### ISSUE TITLE
Image cache not loading

### ISSUE THREAD

**Issue by reidserozi** [localwiki/localwiki#791]
Media/thumbnails images across the platform are not loading.

**Comment by reidserozi** [comment]
![localwiki](https://user-images.githubusercontent.com/3680762/51428603-89797a80-1bd3-11e9-8c75-600f31fde2fa.png)

**Comment by philipn** [comment]
I see the problem on the explore page but is this happening elsewhere? All the pages I pulled up looked fine, eg https://localwiki.org/raleigh/Five_Points

**Comment by reidserozi** [comment]
yes, you are correct. It appears to be the explore page. For example,  https://localwiki.org/raleigh/ and https://localwiki.org/regions

![localwiki issues](https://user-images.githubusercontent.com/3680762/51537755-680bcf00-1e1d-11e9-9288-4903a75f904d.png)

![some interesting problems raleigh](https://user-images.githubusercontent.com/3680762/51537672-27ac5100-1e1d-11e9-83d9-12d6d9e96715.png)
